src/sentry/web/client_config.py

```diff
@@ -122,6 +122,13 @@ def get_client_config(request=None):
     Provides initial bootstrap data needed to boot the frontend application.
     """
     if request is not None:
+        customer_domain = None
+        if hasattr(request, "subdomain") and request.subdomain:
+            customer_domain = {
+                "subdomain": request.subdomain,
+                "organizationUrl": generate_organization_url(request.subdomain),
```

Comment:
for multi-region does organziationUrl still make sense? Or something more generic like "localUrl"

Reply:
@AniketDas-Tekky yes, `organziationUrl` will be needed as this offloads any logic of what the customer domain will look like from the frontend to the backend. 

This is helpful for providing URLs in various environments. For example: 

1. production: `https://orgslug.sentry.io`
2. staging: `https://orgslug.new.staging.getsentry.net/`
3. development: `http://orgslug.dev.getsentry.net:8000/`